src/minesweeper/minesweeper.py

```diff
@@ -12,3 +12,30 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import sys
+import pygame
+from Config import Config
+from ctypes.wintypes import RGB
+
+def runGame():
+    '''This should be in a function called runGame that takes as argument a configuration class 
+    that contains the window size. Use clock.tick() to set the framerate to 60 fps.
+    See the pygame example https://github.com/pygame/pygame/blob/main/examples/chimp.py as well.'''
+
+    pygame.init()
+    pygame.display.set_caption("Minesweeper")
+    config = Config()
```

Comment:
`config` should be an argument of `runGame`, and the initialization should happen at `__init__.py`.